site/docs/v1/tech/apis/_key-generate-post-request-body.adoc

```diff
@@ -14,6 +14,9 @@ The algorithm used to generate the Key. The following values represent algorithm
 * `HS384` - HMAC using SHA-384 hash algorithm
 * `HS512` - HMAC using SHA-512 hash algorithm
 
+[field]#key.issuer# [type]#[String]# [optional]#Optional#::
```

Comment:
There is a default value here, so we need to call it out. Look for other examples where the default is described in the field body.